src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(tab => selectedTabId === tab.id) || tabs[0];
```

Comment:
The `selectedTab` is being determined within the render function. This could potentially lead to inefficient re-calculations on every render. Consider moving this logic to `componentDidUpdate` or `useEffect` (if you're using hooks).